### PR TITLE
Add note on 1K limit for GitHub Actions token

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Using the built-in token only provides 1K requests vs. a user PAT as currently documented.

### What's being changed:

Adding small note to existing rate limit doc, that should probably be word-smithed by someone from the product team.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@nschonni Thanks so much for opening a PR! I'll get this triaged for review

**Comment by skedwards88** [comment]
Thank you for opening this PR to clarify the docs! Can you point me to where you found this rate limit?

**Comment by nschonni** [comment]
https://github.com/tabatkins/bikeshed/runs/2007469198?check_suite_focus=true#step:5:11
Maybe its unique to the GITHUB_TOKEN running as part of a cron job, but I didn't try other combos

**Comment by skedwards88** [comment]
Thank you for opening this issue! I confirmed your finding with the engineering team:

GITHUB_TOKEN is limited to 1000 requests per hour per repository. For organizations that belong to a GitHub Enterprise Cloud account, this limit is 15,000 requests per hour per repository.

You or anyone is welcome to open an issue to update the docs to fix this issue.

**Comment by Odin61** [comment]
Скачайте Outlook для iOS<https://aka.ms/o0ukef>

File: content/rest/overview/resources-in-the-rest-api.md [hunk 1]
```diff
@@ -360,6 +360,8 @@ For users that belong to a {% data variables.product.prodname_ghe_cloud %} accou
 
 {% endif %}
 
+When using the built-in `GITHUB_TOKEN` in GitHub Actions, the rate limit is 1,000.
```
Comment (skedwards88) [inline content/rest/overview/resources-in-the-rest-api.md:363]: Updated with additional info:
```suggestion
When using the built-in `GITHUB_TOKEN` in GitHub Actions, the rate limit is 1,000 requests per hour per repository. For organizations that belong to a GitHub Enterprise Cloud account, this limit is 15,000 requests per hour per repository.
```

**Review by skedwards88 — DISMISSED** [review]
Thank you for opening this issue! I confirmed your finding with the engineering team:

GITHUB_TOKEN is limited to 1000 requests per hour per repository. For organizations that belong to a GitHub Enterprise Cloud account, this limit is 15,000 requests per hour per repository.

I updated your PR with this info. Thank you again for your contribution! 💖 I'll get this merged down for you.

**Comment by nschonni** [comment]
I think the 15K part is covered a few lines above, but happy for you to word-smith as much as you like

**Review by janiceilene — APPROVED** [review]

**Pull request merged**